oeps/processes/oep-0063-proc-toc-arbitration.rst

```diff
@@ -0,0 +1,115 @@
+   
+- *Guide the project toward shared goals*
+- *Help refine and improve plans and designs*
+- *Prevent the project from making bad decisions that are hard to reverse.*
+   
+*The TOC should weigh in:*
+   
+- *on “one-way-doors,” decisions that are hard to reverse, where choices are mutually exclusive*
+- *when the community is having difficulty reaching consensus*
+- *when an important decision is being deferred and not making a call is causing friction*
+- *to focus the community to collaborate on key, shared value, say, investment in maturing LTI*
+
+Specification
+*************
+
+- Requests for arbitration can be posted by anyone in the community - but need to be supported by at least two core contributors or a TOC member
+- The TOC members are encouraged to participate in community discussions early on, before the TOC is asked to arbitrate.
+
+Step-by-step - How to submit an arbitration
+===========================================
+
+- Open a discussion in a public place, such as the `forum <discuss.openedx.org/>`_ or using one of the formal decision recording formats such as `OEPs <https://open-edx-proposals.readthedocs.io/en/latest/>`_. The topics presented need to have been discussed in the community before being formally considered by the TOC.
+- After public review, post a formal arbitration request to the TOC. This can be done in a forum post or a `github ticket <https://github.com/openedx/wg-coordination/issues/new>`_ mentioning the TOC chair (@e0d). See the format below.
+- Core contributors and/or TOC members supporting the arbitration reply in the thread to say so.
+- If the requirements are met, the TOC will schedule the topic for a future meeting. 
+- If the arbitration contains a suggested decision, the TOC can choose to accept the suggested decision, with or without changes, or to refuse it. 
+- The outcome of the arbitration is communicated as a reply to the forum thread of the arbitration, with a comment to explain the decision.
+
+Arbitration - Post template (forum or github)
+=============================================
+
+.. code-block:: none
+
+   Subj: TOC Arbitration - [Title for the arbitration]
+   Category: Community
+   
+   ---------------------
+   
+   ## [Title for the arbitration]
+   
+   ## Summary 
+   [Prior discussion of this topic (with links)]
+   
+   ## Rationale for involving the TOC
+   [Why this topic would benefit from the TOC involvement]
+   
+   ## Decision requested
+   [The precise description of the decision that is requested from the TOC. Note that the TOC doesn't have to take exactly that decision, it can modify it, or refuse it.]
+
+Change History
+**************
+
+2023-05-29
+==========
+
+* Changed the name of the procedure from "appeal" to "arbitration" to address review comments.
```

Comment:
I'm not convinced you need a change history log for changes that occur *within* a single review thread.

Reply:
@sarina What would be the right threshold of changes for adding an history line? You would only add one per pull request that modifies the OEP document?

Reply:
@sarina OK, sounds good - I have removed the additional history in https://github.com/openedx/open-edx-proposals/pull/484/commits/c510773ec784d5adb34268d90de4d0dc210b29f0